docs/source/reference-io.rst

```diff
@@ -181,6 +181,15 @@ Abstract base classes
 .. currentmodule:: trio
 
 
+Converting Async Generators to use streams
```

Comment:
I mean.... I'm not so sure this should be in the `reference-io` file? Like yes, it does use memory channels which is a stream which is I/O, but that's really convoluted. How about updating `reference-core` instead -- maybe even include the function docs instead of linking!

Reply:
yeah that fits better